src/functional.test.ts

```diff
@@ -77,13 +77,9 @@ describe('create-release-branch (functional)', () => {
             },
           });
 
-          expect(await environment.readJsonFile('package.json')).toStrictEqual({
```

Comment:
These tests are updated because there's now an extra `repository` field in `package.json`. That isn't really relevant to these tests, so I've simplified the assertions to only consider `name` and `version`.